### ISSUE TITLE
b3dm 3DTile

### ISSUE THREAD

**Issue by nicholasyum** [egghunters/bim-viewer#25]
support 3DTile to bim-viewer

**Comment by egghunters** [comment]
Yes, it would be fantastic to support 3DTile. It is do-able. @nicholasyum would you like to contribute to this repo? LOL
